src/modules/CRUD/services/CRUDProvider.js

```diff
@@ -190,7 +190,7 @@ export class CRUDProvider extends React.Component {
     }
     const result = {};
     try {
-      const categories = await fetchCustomEndpoint(endpoint);
+      const categories = await fetchCustomEndpoint(`${endpoint}?page_size=999`);
```

Comment:
For your information, I have done a multiple pages fetch to avoid arbitrary page size here -> https://github.com/Terralego/opp-front/blob/e838ec5474ab4357447a0bab9426b5813580739d/src/views/Viewpoint/DetailViewpoint/SidebarTabsContent/InformationTab/index.js#L41

You don't have to do something to resolve this comment, it's just informative.